README.md

```diff
@@ -366,6 +394,24 @@ If you would like to read debug statements on the console, please change `custom
 log4j.logger.ros.local_planner=DEBUG
 ```
 
+## Safe Landing Planner
+
+Once the catkin workspace has been built, to run the planner with a Realsense D435 and Occipital Structure Core, you can generate the launch file using the script *generate_launchfile.sh*. The script works the same as described in the section above for the Local Planner. For example:
+
+```bash
+export CAMERA_CONFIGS="camera_main,struct_core,819612070807,0.3,0.32,-0.11,0,0,0"
+export VEHICLE_CONFIG_SLP=~/catkin_ws/src/avoidance/safe_landing_planner/cfg/slpn_structure_core.yaml
+export VEHICLE_CONFIG_WPG=~/catkin_ws/src/avoidance/safe_landing_planner/cfg/wpgn_structure_core.yaml
```

Comment:
Does the waypoint generator need to be aware of the camera type? Shouldn't that be handled by the planner part?